### PR TITLE
feature/release-script

### PR DESCRIPTION
**Pull request by seesleestak**
First pass at a release script for #27 . You can run it two ways:
1. Provide the release type inline (major, minor, patch)
```
npm run release minor
```
2. No arguments, get a prompt

### CODE REVIEW

**Comment by seesleestak** [comment]
Also, I haven't run it all the way through yet since I didn't feel like unwinding stuff.

**Comment by seesleestak** [comment]
Probably wouldn’t hurt to have colors to make some of the messages easier to understand. Figured it wasn’t required 🤷🏼‍♂️

File: scripts/release.sh [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  echo "Please select release type:"
+  echo "   1) major ($MAJOR_VERSION)"
+  echo "   2) minor ($MINOR_VERSION)"
+  echo "   3) patch ($PATCH_VERSION)"
+  read RESPONSE
+  get_version $RESPONSE
+fi
+echo "Creating a $VERSION_TYPE ($NEW_VERSION) release"
+
+stash_checkout_version
```
Comment (billyxs) [inline scripts/release.sh]: Great script man! I'm good with all this. 

Only thought I had was that where the pull from master occurs is after the current version is read. If master isn't up to date, or the branch current checked out has a stale version, the version created could be incorrect. Totally fine to leave it as is.

**Comment by billyxs** [comment]
> Probably wouldn’t hurt to have colors to make some of the messages easier to understand. Figured it wasn’t required 🤷🏼‍♂️

always a fan of colors

File: scripts/release.sh (hunk 1 above)
Comment (seesleestak) [reply scripts/release.sh]: Ahh good point. Will find a fix for that tomorrow.

**Comment by seesleestak** [comment]
You want me to try a release after merging it? Major? Minor?

**Comment by billyxs** [comment]
> You want me to try a release after merging it? Major? Minor?

Awesome man! Yeah, let's do a patch or minor release, and if all is going well, let's do a 1.0 major release soon.

**Comment by seesleestak** [comment]
Sounds good, I'll give it a whirl

**Pull request merged**